src/methods/shared_docstrings.jl

```diff
@@ -125,3 +125,82 @@ const CHECKMEMORY_KEYWORD = """
 - `checkmemory`: If `true` (the default), check if there is enough memory for the operation. 
     `false` will ignore memory needs.
 """
+
+const SCALE_KEYWORD = """
+- `scale`: set `scale` for `x * scale + offset` transformations. 
+"""
+
+const OFFSET_KEYWORD = """
+- `offset`: set `offset` for `x * scale + offset` transformations. 
+"""
+
+const RAW_KEYWORD = """
+- `raw`: Turn of all scaling and masking and load the raw values from disk.
+    `false` by default. If `true`, `scaled` will be set to `false` and `maskingval`
+    will be set to `nothing`. A warning will be printed if `scaled` or `maskingval`
+    are manually set to another value.
+"""
+
+const SCALED_KEYWORD = """
+- `scaled`: apply scale and offset as `x * scale + offset`. `true` by default.
+    This is common where data has been convert to e.g. UInt8 to save disk space.
+    To ignore `scale` and `offset` metadata, use `scaled=false`. If `scale` and
+    Note: `offset` are `1.0` and `0.0` they will be ignored and the original type will 
+    be used even when `scaled=true`. This is because these values may be fallback 
+    defaults and we do not want to convert every `Real` array to larger `Float64` values.
+"""
+
+const COERCE_KEYWORD = """
+- `coerce`: where `scale` and/or `offset` are present during `setindex!` to disk, 
+    coerce values to the disk type. `convert` is the default, but `round`, `trunc` or
+    or `ceil` may be needed where the values are not exact.
+"""
+
+const MISSINGVAL_KEYWORD = """
+- `missingval`: value representing missing data, normally detected from the file. Set manually
+    when you know the value is not specified or is incorrect. This will *not* change any
+    values in the raster, it simply assigns which value is treated as missing. 
+"""
+
+const MASKINGVAL_KEYWORD = """
+- `maskingval`: A value to convert `missingval` to, by default `missing`. If this is set it 
+    will be the return value of `missingval(raster)` - `maskingval` becomes the new `missingval`.
+    Setting `maskingval` to `nothing` means no masking will occur, and the original `missingval` 
+    will be the final `missingval`. This can give better performance than using `missing`. 
+    Another efficient option is to use e.g. `zero(eltype(raster))` to replace missing values with zero.
```

Comment:
I can see why you picked this but I'm not totally happy about the name.  Still, I can't see any other nice options.  `missingval` would have been nice but it's already in use.  I'd say we should go for it.  A sufficiently good docstring will clarify all problems :P 

Maybe emphasize that this does what `replace_missing` does?  
```suggestion
- `maskingval`: A value to convert `missingval` to, by default `missing`. If this is set it 
    will be the return value of `missingval(raster)` - `maskingval` becomes the new `missingval`.
    This is the same thing that `replace_missing(raster, some_value)` does.
    Setting `maskingval` to `nothing` means no masking will occur, and the original `missingval` 
    will be the final `missingval`. This can give better performance than using `missing`. 
    Another efficient option is to pass e.g. `zero(eltype(raster))` to replace missing values with zero.
```

Reply:
It's also what CommonDataModel uses so there's precedent... But sharing the mask part with the `mask` function is kinda weird for me

Reply:
I've decided to scrap `maskingval` and just have `missingval`, but allow `sourcemissingval => destmissingval` as a syntax

It's a bit more breaking but I think nicer long term